src/client/mod.rs

```diff
@@ -73,6 +75,11 @@ impl ClientBuilder {
         self
     }
 
+    pub fn backoff_config(mut self, backoff_config: BackoffConfig) -> Self {
+        self.backoff_config = Some(backoff_config);
+        self
+    }
```

Comment:
Public methods should have a doc string

`/// Set configuration for retrying requests to the broker`, or something along those lines.

Reply:
"Set up" isn't really correct. `Set` if anything.